test/python/tensorflow/tf_unittest_runner.py

```diff
@@ -206,5 +212,24 @@ def read_tests_from_file(filename):
         ]
 
 
+def print_results(test_list, test_result):
+    succeeded = []
+    failures = []
+    errors = []
+    for test in test_list:
+        if test_result.wasSuccessful():
+            succeeded.append(test)
+        elif test_result.failures:
+            failures.append(test)
+        elif test_result.errors:
```

Comment:
`test_result.failures` and `test_result.errors` are booleans? The names `failures` and `errors` sound like they are lists or somesuch.

@sindhu-nervana can you please confirm the types of `test_result.failures` and `test_result.errors`

Reply:
Yes, '''test_result.failures''' and '''test_result.errors''' are lists and TestResult object stores these.
A list containing 2-tuples of TestCase instances and strings holding formatted tracebacks.

Reply:
If they are lists, we should convert them to booleans explicitly
maybe something like:
`len(test_result.failures)>0`